lib/buyers/tests/test_api.py

```diff
@@ -41,16 +43,26 @@ def test_add_multiple(self):
         eq_(self.get_errors(res.content, 'uuid'),
             [BUYER_UUID_ALREADY_EXISTS])
 
-    def test_add_empty(self):
+    def test_uuid_cannot_be_empty(self):
```

Comment:
I don't usually like to make extraneous changes in a patch but I whenever I read one of andy's test names and can't figure it out I spend time to figure it out then rename the test ;)